components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/NotificationConstants.java

```diff
@@ -18,13 +18,25 @@
 
 package org.wso2.carbon.identity.event.handler.notification;
 
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
 public class NotificationConstants {
 
     private NotificationConstants() {
     }
 
+    private static String getClaimUriLocale() {
+
+        if (Boolean.parseBoolean(IdentityUtil.getProperty("useLegacyLocalizationClaim"))
+        || IdentityUtil.getProperty("useLegacyLocalizationClaim") == null) {
```

Comment:
if IdentityUtil.getProperty("useLegacyLocalizationClaim") == null we need to return "http://wso2.org/claims/local"

Reply:
Yes. if it is not defined, we have to use the new behavior.

Reply:
We need to check the behavior in challenge questions too. Please check that too.